exporter/datadogexporter/config_warnings.go

```diff
@@ -32,13 +32,6 @@ var renamedSettings = []deprecatedError{
 			c.Metrics.HistConfig.SendAggregations = c.Metrics.HistConfig.SendCountSum
 		},
 	},
-	{
-		oldName: "traces::peer_service_aggregation",
-		newName: "traces::peer_tags_aggregation",
-		updateFn: func(c *Config) {
-			c.Traces.PeerTagsAggregation = c.Traces.PeerServiceAggregation
-		},
-	},
```

Comment:
If removing this option, we should add this to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f7cd7348b6afd0fc4cb0a65651e8e4efbfc42193/exporter/datadogexporter/config.go#L532

I don't understand why we are doing this on this PR, though